Firebase/Messaging/FIRMessagingClient.m

```diff
@@ -16,8 +16,13 @@
 
 #import "FIRMessagingClient.h"
 
-#import <FirebaseCore/FIRReachabilityChecker.h>
```

Comment:
This change shouldn't be needed. copybara will make the transformation for outside of CocoaPods.